yoti_python_sdk/doc_scan/session/create/check/document_authenticity.py

```diff
@@ -41,7 +59,44 @@ class RequestedDocumentAuthenticityCheckBuilder(object):
     Builder to assist creation of :class:`RequestedDocumentAuthenticityCheck`
     """
 
-    @staticmethod
-    def build():
-        config = RequestedDocumentAuthenticityCheckConfig()
+    def __init__(self):
+        self.__manual_check = None
+
+    def with_manual_check_always(self):
+        """
+        Sets the value of manual check to "ALWAYS"
+
+        :return: the builder
+        :rtype: RequestedDocumentAuthenticityCheckBuilder
+        """
+        self.__manual_check = constants.ALWAYS
+        return self
+
+    def with_manual_check_fallback(self):
+        """
+        Sets the value of manual check to "FALLBACK"
+
+        :return: the builder
+        :rtype: RequestedDocumentAuthenticityCheckBuilder
+        """
+        self.__manual_check = constants.FALLBACK
+        return self
+
+    def with_manual_check_never(self):
+        """
+        Sets the value of manual check to "NEVER"
+
+        :return: the builder
+        :rtype: RequestedDocumentAuthenticityCheckBuilder
+        """
+        self.__manual_check = constants.NEVER
+        return self
+
+    def build(self=None):
+        if self is None:
+            manual_check = None
+        else:
+            manual_check = self.__manual_check
```

Comment:
Previously `@staticmethod` which we need to support for backwards compatibility (we can remove this in the major release)

Reply:
This cannot work with Python 2, so we need to deprecate this builder and introduce a new one